### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by nbirillo**
Sorry, I deleted the windows workflow because `KtAnnotated.annotationEntries()` always returns an empty list in Windows. We can replace `\r\n` to `\n` manually in tests if the currents OS is Windows or we can test it later (we should find a way how we can to set the compiler). What do you think? I prefer the second option now.

This PR contains only tests for objects

### CODE REVIEW

**Comment by isomethane** [comment]
Did we have problems with \n in Windows?

**Comment by nbirillo** [comment]
Yes, it is a problem in Windows. We should find a way how we can set up the kotlin compiler. Now it works incorrectly with annotations in Windows

**Comment by isomethane** [comment]
I still don't understand. Do you think that compiler works incorrectly because of line breaks? Are there any other problems with them?

**Comment by nbirillo** [comment]
The compiler expected \n, but for Windows it is not true. It gets \r\n. The default settings of compiler considered only UNIX new line symbols. To change it we should find a way to set it, but I am not sure that we should do it in this PR, because we can do it later

**Comment by isomethane** [comment]
Thanks you, I totally agree.

File: reflekt-plugin/src/test/kotlin/io/reflekt/plugin/analysis/ReflektUsesTestModel.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.jetbrains.kotlin.psi.KtNamedFunction
+
+/*
+* There are helper data classes to parse JSON.
+* TODO: I am not sure that it is the best way, but I did not find a better way.
+* */
+
+data class SubtypesToUsesTest(
+    val subtypes: Set<String> = emptySet(),
+    val uses: MutableList<String> = mutableListOf()
```
Comment (isomethane) [inline reflekt-plugin/src/test/kotlin/io/reflekt/plugin/analysis/ReflektUsesTestModel.kt:12]: Can't we just use MutableSet<String> here so we don't need to write functions to compare lists as sets?

**Review by isomethane — APPROVED** [review]

**Pull request merged**